src/azure-cli-telemetry/azure/cli/telemetry/components/records_collection.py

```diff
@@ -52,7 +52,7 @@ def snapshot_and_read(self):
             if stat.S_ISREG(each[1].st_mode):
                 try:
                     # Platform question: if this op is atom
-                    os.rename(os.path.join(folder, each[0]), os.path.join(tmp, each[0]))
+                    shutil.move(os.path.join(folder, each[0]), os.path.join(tmp, each[0]))
```

Comment:
this is an existing external dependency right, just want to double confirm

Reply:
It's from Python library and already imported in this file.